boards/arm/adafruit_feather_stm32f405/pinmux.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020 Lucian Copeland for Adafruit Industries
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <sys/sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* pin assignments for Feather STM32F405 board */
+static const struct pin_config pinconf[] = {
+#ifdef CONFIG_UART_3
+	{STM32_PIN_PB10, STM32F4_PINMUX_FUNC_PB10_USART3_TX},
+	{STM32_PIN_PB11, STM32F4_PINMUX_FUNC_PB11_USART3_RX},
+#endif /* CONFIG_UART_3 */
+#ifdef CONFIG_I2C_1
+	{STM32_PIN_PB6, STM32F4_PINMUX_FUNC_PB6_I2C1_SCL},
+	{STM32_PIN_PB7, STM32F4_PINMUX_FUNC_PB7_I2C1_SDA},
+#endif /* CONFIG_I2C_1 */
+#ifdef CONFIG_SPI_2
+	{STM32_PIN_PB13, STM32F4_PINMUX_FUNC_PB13_SPI2_MASTER_SCK},
+	{STM32_PIN_PB14, STM32F4_PINMUX_FUNC_PB14_SPI2_MASTER_MISO},
+	{STM32_PIN_PB15, STM32F4_PINMUX_FUNC_PB15_SPI2_MASTER_MOSI},
+#endif /* CONFIG_SPI_2 */
```

Comment:
`STM32F4_PINMUX_FUNC_PINFOO_SPI2_MASTER` are new macros that need to be defined.
Though, why not using the version w/o _MASTER_ that are already available?

Reply:
Sorry, that's just a typo on my part, I didn't notice the naming convention was different.